content/index.rst

```diff
@@ -12,42 +12,86 @@ available in the computer that you will use during the workshop.
 
    - `Aalto University <https://scicomp.aalto.fi/aalto/coderefinery-setup/>`__
 
+.. admonition:: Important: don't start installing yet for the 2024-03 workshop
+   :class: warning
+
+   We are still revising our instructions to make the whole workshop
+   easier.  **We recommend you wait until the week before to start
+   following this page.**
+
 
 Requirements table
 ------------------
 
 First, go through all the items in the check list below. If you are in doubt or
 your answer is "no", then follow the linked guidelines for each.
+First, ask how you want to do the exercises:
+
+* **web** uses only the Github web interface.  This is simple, but not
+  very practical for day to day work. *Recommended if you are trying
+  it out, but not doing development yourself yet*
+* **VSCode** uses the popular VSCode editor.  It makes things easier,
```

Comment:
Should VSCode mentions/instructions be left out from here until the first weeks tools are decided?

Reply:
True, it's not decided.  Would you say it's likely?  Or could mentioning it (but not promising) encourage more people to sign up?

I'll edit.